inc/Engine/Admin/Settings/Render.php

```diff
@@ -314,6 +332,21 @@ public function number( $args ) {
 	 * @param array $args Array of arguments to populate the template.
 	 */
 	public function categorized_multiselect( $args ) {
+		if ( is_array( $args['value'] ) ) {
+			$args['value'] = implode( "\n", $args['value'] );
+		}
+
+		$args['value'] = empty( $args['value'] ) ? '' : $args['value'];
+		$args['items'] = empty( $args['items'] ) ? new stdClass() : $args['items'];
+
+		$wp_rocket_textarea = get_rocket_option( esc_attr( $args['id'] ) );
```

Comment:
```suggestion
		$wp_rocket_textarea = get_rocket_option( sanitize_key( $args['id'] ) );
```

Reply:
Oh, I believe we don't need this part anyway because I split them into two fields